Microsoft.Toolkit.Uwp/Helpers/BluetoothLEHelper/ObservableBluetoothLEDevice.cs

```diff
@@ -0,0 +1,456 @@
+        {
+            DeviceInfo = deviceInfo;
+            Name = DeviceInfo.Name;
+
+            IsPaired = DeviceInfo.Pairing.IsPaired;
+
+            LoadGlyph();
+        }
+
+        /// <summary>
+        /// Gets the bluetooth device this class wraps
+        /// </summary>
+        /// <value>The bluetooth le device.</value>
+        public BluetoothLEDevice BluetoothLEDevice
+        {
+            get
+            {
+                return _bluetoothLeDevice;
+            }
+
+            private set
+            {
+                _bluetoothLeDevice = value;
+                OnPropertyChanged();
+            }
+        }
+
+        /// <summary>
+        /// Gets or sets the glyph of this bluetooth device
+        /// </summary>
+        /// <value>The glyph.</value>
+        public BitmapImage Glyph
+        {
+            get
+            {
+                return _glyph;
+            }
+
+            set
+            {
+                _glyph = value;
+                OnPropertyChanged();
+            }
+        }
+
+        /// <summary>
+        /// Gets the device information for the device this class wraps
+        /// </summary>
+        /// <value>The device information.</value>
+        public DeviceInformation DeviceInfo
+        {
+            get
+            {
+                return _deviceInfo;
+            }
+
+            private set
+            {
+                _deviceInfo = value;
+                OnPropertyChanged();
+            }
+        }
+
+        /// <summary>
+        /// Gets a value indicating whether this device is connected
+        /// </summary>
+        /// <value><c>true</c> if this instance is connected; otherwise, <c>false</c>.</value>
+        public bool IsConnected
+        {
+            get
+            {
+                return _isConnected;
+            }
+
+            private set
+            {
+                if (_isConnected != value)
+                {
+                    _isConnected = value;
+                    OnPropertyChanged();
+                }
+            }
+        }
+
+        /// <summary>
+        /// Gets a value indicating whether this device is paired
+        /// </summary>
+        /// <value><c>true</c> if this instance is paired; otherwise, <c>false</c>.</value>
+        public bool IsPaired
+        {
+            get
+            {
+                return _isPaired;
+            }
+
+            private set
+            {
+                if (_isPaired != value)
+                {
+                    _isPaired = value;
+                    OnPropertyChanged();
+                }
+            }
+        }
+
+        /// <summary>
+        /// Gets the services this device supports
+        /// </summary>
+        /// <value>The services.</value>
+        public ObservableCollection<ObservableGattDeviceService> Services
+        {
+            get
+            {
+                return _services;
+            }
+
+            private set
```

Comment:
No need to expose the private set if it's not being used. It's the same instance all the time.